components/ClientRedirect.tsx

```diff
@@ -11,24 +11,30 @@ export function ClientRedirect({ url }) {
 
   return (
     <>
+      <style jsx global>
+        {`
+          .redirect-page {
+            color: #383838;
+            font-family: 'Menlo', 'Monaco', Courier, monospace;
+            font-size: 0.8em;
+            display: flex;
+            height: 90vh;
+            justify-content: center;
+            align-items: center;
+          }
+          .dark-mode .redirect-page {
```

Comment:
Support dark mode for redirect page. Preview:

![image](https://user-images.githubusercontent.com/7823011/196802641-5a6b2aeb-9aa7-42a8-9dde-aebb2f763966.png)